### PR TITLE
Assets stability

### PR DESCRIPTION
**Pull request by antonyagustine**
Ticket - **CS-30006**

- Asset module improvements in import and export command

### CODE REVIEW

**Review by netrajpatel — APPROVED** [review]

File: packages/contentstack-export/src/app.js [hunk 1]
```diff
@@ -126,9 +127,11 @@ const allExport = async (config, types, branchName) => {
     const stackClient = stack
       .Client(config)
       .stack({ api_key: config.source_stack, management_token: config.management_token });
-    for (const type of types) {
-      const ExportModule = require('./lib/export/' + type);
+
```
Comment (shafeeqd959) [inline packages/contentstack-export/src/app.js:130]: Please check the extension prettier and its configuration , make sure it is enabled

File: packages/contentstack-export/src/lib/export/assets.js [hunk 2]
```diff
@@ -75,212 +79,223 @@ ExportAssets.prototype = {
...
-                            config,
-                            chalk.red('The following asset failed to download\n' + JSON.stringify(assetJSON)),
-                          );
-                          addlogs(config, error, 'error');
-                        });
+                      if (self.assetConfig.downloadVersionAssets) {
+                        return self
+                          .getVersionedAssetJSON(assetJSON.uid, assetJSON._version)
+                          .then(function () {
+                            self.assetContents[assetJSON.uid] = assetJSON;
```
Comment (shafeeqd959) [inline packages/contentstack-export/src/lib/export/assets.js:86]: Please make use of the latest concurrency utility

File: packages/contentstack-export/src/lib/export/assets.js [hunk 3]
```diff
@@ -75,212 +79,223 @@ ExportAssets.prototype = {
...
+
+  getAssetCount(folder) {
+    const self = this
     return new Promise(function (resolve, reject) {
       if (folder && typeof folder === 'boolean') {
-        let queryOptions = { include_folders: true, query: { is_dir: true }, include_count: true };
-        client
-          .stack({ api_key: config.source_stack, management_token: config.management_token })
+        let queryOptions = {
+          skip: 99999990,
```
Comment (shafeeqd959) [inline packages/contentstack-export/src/lib/export/assets.js:214]: Why skip is 99999990 ?

**Comment by shafeeqd959** [comment]
@antonyagustine , please have a look into entries, content types in export to understand the use of concurrent utility and file utility updates. Please let me know if you need a call 

CC: @netrajpatel

File: packages/contentstack-export/src/lib/export/assets.js (hunk 3 above)
Comment (antonyagustine) [reply packages/contentstack-export/src/lib/export/assets.js:214]: This method is meant to get only counts so we doing max/large number of skip to skip fetching content and trying to get only count

File: packages/contentstack-export/src/lib/export/assets.js (hunk 2 above)
Comment (antonyagustine) [reply packages/contentstack-export/src/lib/export/assets.js:86]: Hi Shafeew, Asper our call this morning I'm skipping this point

**Pull request merged**